lib/main.js

```diff
@@ -10,6 +10,7 @@ let events = require("sdk/system/events");
 let runtime = require("sdk/system/runtime");
 let utils = require("sdk/window/utils");
 let system = require("sdk/system");
+let _ = require("sdk/l10n").get;
```

Comment:
According to "Using Localized Strings in JavaScript" section in [MDN localization tutorial](https://developer.mozilla.org/en-US/Add-ons/SDK/Tutorials/l10n) this is done for compatibility with existing translation tools, like gettext. The strings are few, so there is no need for special localization tools, but I still used this convention because all examples are written this way. If you prefer I will change _(...) to l10n.get(...).